virttest/bootstrap.py

```diff
@@ -894,6 +894,4 @@ def bootstrap(options, interactive=False):
     logging.info("")
     step += 1
     logging.info("%d - If you wish, you may take a look at the online docs for "
-                 "more info", step)
-    logging.info("")
-    logging.info(online_docs_url)
+                 "more info: %s", step, online_docs_url)
```

Comment:
This one is missing:

```diff
diff --git a/virttest/bootstrap.py b/virttest/bootstrap.py
index 74b1e5d..0f51a5e 100644
--- a/virttest/bootstrap.py
+++ b/virttest/bootstrap.py
@@ -278,8 +278,8 @@ def sync_download_dir(interactive):
             diff_result = process.run(
                 diff_cmd, ignore_status=True, verbose=False)
             if diff_result.exit_status != 0:
-                logging.info("%s result:\n %s",
-                             diff_result.command, diff_result.stdout)
+                logging.debug("%s result:\n %s",
+                              diff_result.command, diff_result.stdout)
                 answer = genio.ask('Download file "%s" differs from "%s". '
                                    'Overwrite?' % (dst_file, src_file),
                                    auto=not interactive)
```

And the diff looks bad all in blue (when the next commit is applied).